src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#[cfg(test)]
```

Comment:
I'm actually a little proud of that submodule purge above, it's more difficult than you think :D 


This `lib.rs` file is probably pointless, but I think we'd set up a crate here anyway at some point.

Reply:
(it makes no sense in the main PR view, click "View changes")

Reply:
Yea... not sure how that submodule happened. Probably I did it by accident somehow.